tools/paas_dashboard/config.ru

```diff
@@ -0,0 +1,21 @@
+require 'dashing'
+
+configure do
+  set :auth_token, 'YOUR_AUTH_TOKEN'
```

Comment:
It's used to secure requests to the API that provides a push interface for updating data bound to widgets. We don't use the push mechanism at the moment, and we don't need to secure it, so I left it as the default value.